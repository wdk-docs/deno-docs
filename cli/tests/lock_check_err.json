{
  "http://127.0.0.1:4545/cli/tests/subdir/print_hello.ts": "5c93c66125878389f47f4abcac003f4be1276c5223612c26302460d71841e287",
  "http://127.0.0.1:4545/cli/tests/003_relative_import.ts": "bad"
}
