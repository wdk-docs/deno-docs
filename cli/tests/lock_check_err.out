[WILDCARD]Subresource integrity check failed --lock=lock_check_err.json
http://127.0.0.1:4545/cli/tests/003_relative_import.ts
